modules.d/90kernel-modules/module-setup.sh

```diff
@@ -128,6 +128,11 @@ installkernel() {
         hostonly='' instmods "=crypto"
         instmods "=arch/$arch/crypto" "=drivers/crypto"
     fi
+
+    inst_multiple -o "$depmodd/*.conf"
```

Comment:
You need to change this to only include configuration files that we ship since dont blindly include everything in directories *except* for host only setups in which the administrator is expected to know what he or she is doing and what is placed there.

Reply:
If some distro is shipping something in `/usr/lib/depmod.d` , then it has a reason and we better use it.